TechnicalDocuments/ToolsAndServices.md

```diff
@@ -0,0 +1,28 @@
+# Tools and Services
+
+These are tools and services used by the BabylonHealth's Consumer Apps tribe.
```

Comment:
There are others, working in different tribes, that would still use these tools.